src/app/components/provenances/provenances.menus.ts

```diff
@@ -0,0 +1,71 @@
+import { RouterStateSnapshot } from "@angular/router";
+import { retrieveResolvedModel } from "@baw-api/resolver-common";
+import { Category, menuRoute } from "@interfaces/menusInterfaces";
+import { AudioEventProvenance } from "@models/AudioEventProvenance";
+import {
+  defaultEditIcon,
+  defaultNewIcon,
+  isAdminPredicate,
+} from "src/app/app.menus";
+import { CommonRouteTitles } from "src/app/stringConstants";
+import { provenanceRoute, provenancesRoute } from "./provenances.routes";
+
+/*
+  Provenances Category
+*/
+export const provenancesCategory: Category = {
+  label: "Provenances",
+  icon: ["fas", "fingerprint"],
+  route: provenancesRoute,
+};
+
+export const provenancesMenuItem = menuRoute({
+  icon: ["fas", "fingerprint"],
+  label: "Provenances",
+  route: provenancesRoute,
+  tooltip: () => "View provenances",
+  predicate: isAdminPredicate,
+});
+
+export const newProvenanceMenuItem = menuRoute({
+  icon: defaultNewIcon,
+  label: "New provenance",
+  parent: provenancesMenuItem,
+  predicate: isAdminPredicate,
+  route: provenancesRoute.add("new"),
+  tooltip: () => "Create a new provenance",
+});
+
+/*
+  Provenance Category
+*/
+
+export const provenanceCategory: Category = {
+  label: "Provenance",
+  icon: provenancesCategory.icon,
+  route: provenanceRoute,
+};
+
+export const provenanceMenuItem = menuRoute({
+  icon: ["fas", "fingerprint"],
+  label: "Provenance",
+  parent: provenancesMenuItem,
+  route: provenanceRoute,
+  tooltip: () => "The current provenance",
```

Comment:
Note to self: A lot of these tooltips are bad, and I'll probably need to manually change them